packages/ember-glimmer/lib/utils/references.js

```diff
@@ -52,6 +52,27 @@ export class UpdatableReference extends RootReference {
   }
 }
 
+export class GetHelperReference {
+  constructor(args) {
+    this.positionalArgs = args.positional;
+  }
+
+  isDirty() { return true; }
+
+  value() {
+    let objRef = this.positionalArgs.at(0).value(),
+      pathRef = this.positionalArgs.at(1).value();
+
+    return objRef ? get(objRef, pathRef) : objRef;
```

Comment:
Fullfils [should handle object values as nulls](https://github.com/emberjs/ember.js/pull/13173/files#diff-350c933360e807598f74de6573a7fcaaR189) test.

Should it return `null` for any falsy `objRef` value?